src/test/run-pass/intrinsics-math.rs

```diff
@@ -21,39 +21,100 @@ macro_rules! assert_approx_eq {
 
 mod rusti {
     extern "rust-intrinsic" {
+        #[cfg(stage0)]
```

Comment:
Can you update this test to not use the intrinsics at all? It's actually best to go through the standard library anyway to deal with linkage weirdness.